src/hooks/useNavigatePage.ts

```diff
@@ -179,6 +182,56 @@ export const useNavigatePage = () => {
     [processTasks],
   );
 
+  const getCurrentPageIndex = () => {
+    const location = window.location.href;
+    const _currentPageId = location.split('/').slice(-1)[0];
+    return order?.indexOf(_currentPageId) ?? -1;
```

Comment:
I always prefer using `undefined` as the default return value. Returning `-1` means that the typescript return hint shows as `number`, and checking it with `typeof returnValue === 'number'` always succeeds - and you'd have to look at the implementation to know what the error value even is.

Reply:
Yeah, I agree. I'll change the default to `undefined`, and change it at all the places it was used as well